packages/dds/tree/src/tableSchema.ts

```diff
@@ -225,6 +219,10 @@ export namespace System_TableSchema {
 				for (const row of tableNode.rows) {
 					const cell = row.getCell(this.id);
 					if (cell !== undefined) {
+						if (!Tree.is(cell, cellSchema)) {
+							throw new UsageError("Parent table contains a cell with unexpected schema.");
```

Comment:
Since the cell is presumably allowed in the table, just not this column:
```suggestion
							throw new UsageError("Parent table contains a cell with unexpected schema.");
```
```suggestion
							throw new UsageError("Parent table contains a cell with incompatible with this column.");
```